addons/docs/package.json

```diff
@@ -71,6 +71,7 @@
     "js-string-escape": "^1.0.1",
     "lodash": "^4.17.15",
     "prop-types": "^15.7.2",
+    "sveltedoc-parser": "^3.0.4",
```

Comment:
add to `@storybook/svelte` and make it an optional peer dependency of `@storybook/addon-docs`, see `vue-docgen-loader` for an example